ccloud/README.md

```diff
@@ -59,6 +71,15 @@ It provides client examples written in various programming languages.
 
 ![image](../clients/cloud/images/clients-all.png)
 
+
+# Build Your Own Cloud Demo
+
+As a next step, you may want to build your own custom demo or test environment.
+We have a Docker Compose file called [cp-all-in-one-cloud](../cp-all-in-one-cloud/README.md) -- use this with your pre-configured Confluent Cloud instance.
```

Comment:
Instead of 
> pre-configured

Do you mean?
> existing

The user will have to do some configuration w/ cp-all-in-one-cloud so maybe "existing" is more accurate.

Reply:
Ack and done